tests/testCommonFunctions/testCommonFunctions.m

```diff
@@ -323,6 +342,23 @@ function testGroupLayersMod(testCase)
             outputs = {out1,out2};
             testCase.verifyEqual(testCase.groupLayersModOutputs,outputs, 'RelTol', testCase.tolerance, 'AbsTol', testCase.abs_tolerance);
         end
+
+        function testGroupLayersImaginaryMod(testCase)
+            % testGroupLayersModImaginary
+            %
+            % Inputs:
+            %   testCase - class instance
+            %
+            % Outputs:
+            %   none
+            %
```

Comment:
Not sure how useful these comments are, should we leave them in?

Reply:
I agree, let's remove them.